src/Illuminate/Contracts/Logging/ChannelResolutionException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Illuminate\Contracts\Logging;
+
+use Exception;
+
+class ChannelResolutionException extends Exception
```

Comment:
Why are you defining concrete classes inside the Contracts namespace?

Reply:
Sorry, that's a mistake, thanks